autocompleter/base.py

```diff
@@ -157,6 +160,36 @@ def get_old_terms(cls, obj_id):
             old_terms = cls._deserialize_data(old_terms)
         return old_terms
 
+    @classmethod
+    def delete_old_facets(cls, obj_id, old_facets):
```

Comment:
for consistency call old_facets => facts or in clear_keys call norm_terms old_norms_terms ... no preference which....